packages/neuron-wallet/src/models/address.ts

```diff
@@ -19,6 +19,6 @@ export interface Address {
   balance?: string
   version?: AddressVersion
   description?: string
-  isImporting?: boolean | undefined
-  usedByAnyoneCanPay?: boolean | undefined
+  isImporting?: boolean
```

Comment:
https://github.com/nervosnetwork/neuron/blob/4de122604aef9f5d13f593a6344061994022aa38/packages/neuron-wallet/src/database/address/meta.ts#L23-L24
Here also can be remove `undefined`.

Reply:
Removed by [bc1d49f](https://github.com/Magickbase/neuron/pull/52/commits/bc1d49f752d48f8d334339dbb2493f417f2c5c01)